### PR TITLE
add edit button to project detail page,

### PR DESCRIPTION
**Pull request by vellip**
rename rule to be conform with naming convention,
make follow button in span instead for inline display

### CODE REVIEW

File: euth/projects/templates/euth_projects/includes/project_hero_unit.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
-
-{% load thumbnail react_follows project_tags i18n %}
+{% load thumbnail react_follows project_tags i18n  rules%}
```
Comment (slomo) [inline euth/projects/templates/euth_projects/includes/project_hero_unit.html]: Nitpic: Space are odd here.

File: euth/projects/templates/euth_projects/includes/project_hero_unit.html [hunk 2]
```diff
@@ -28,8 +27,16 @@ <h1 class="herounit-title">{{ project.name }}</h1>
...
-                {% block phase_cta %}{% endblock %}
+                {% has_perm 'euth_projects.edit_project' user project as can_edit_project %}
+                {% if can_edit_project %}
+                    <a href="{% url 'dashboard-project-edit' project.organisation.slug project.slug %}" class="btn btn-dark btn-primary btn-sm">
+                        <i class="fa fa-edit"></i> {% trans 'edit' %}
+                    </a>
+                {% endif %}
+
                 {% react_follows project %}
+                {% load rules %}
```
Comment (slomo) [inline euth/projects/templates/euth_projects/includes/project_hero_unit.html]: ???

**Comment by slomo** [comment]
Works!

**Comment by vellip** [comment]
Whoops, is pretty now.

**Pull request merged**